osu.Game/Online/Chat/ChannelManager.cs

```diff
@@ -133,7 +134,9 @@ public void OpenPrivateChannel(APIUser user)
 
         private void currentChannelChanged(ValueChangedEvent<Channel> e)
         {
-            if (!(e.NewValue is ChannelSelectorTabItem.ChannelSelectorTabChannel))
+            bool isSelectorChannel = e.NewValue is ChannelSelectorTabItem.ChannelSelectorTabChannel || e.NewValue is DummySelectorChannel;
```

Comment:
`ChannelSelectorTabItem.ChannelSelectorTabChannel` is being used in the current chat overlay.
I wanted to make sure I didn't break any existing behaviour when making this change.
Checking against it will be removed when the new chat overlay replaces the old one.

Reply:
Right, thanks.